Presentation/Program.cs

```diff
@@ -0,0 +1,173 @@
+﻿using Simple_Inventory_Management_System.Services;
+
+internal class Program
```

Comment:
```suggestion
public class Program
```

the access modifier for the class shouldn't be less than the one for its members.